### PR TITLE
Update webpack.config.js

### PR DESCRIPTION
**Pull request by realmunk**
Support symlinks

### CODE REVIEW

**Comment by espen42** [comment]
A good idea as such. It makes me a little reluctant to pull this into the default functionality, when webpack themselves use `symlinks: true` as their default. 

I suggest a different approach (at least until I know more about the symlinks setting): take a look at today's update 0.2.9 for starter-react4xp. It bumps some versions in `package.json` (`"react4xp-build-components": "^0.3.10"` and `"react4xp-buildconstants": "^0.7.7"`), and build.gradle updates `lib-react4xp` to `0.3.8` along with [several changes to the build logic](https://github.com/enonic/starter-react4xp/blob/774adf03d32fb7722ada4b4e642de241fcba45f0/build.gradle). **The outcome of this is two things**: 
- The overrides that were previously a JSON string, now live in a separate `properties` file: [react4xp.properties](https://github.com/enonic/starter-react4xp/blob/774adf03d32fb7722ada4b4e642de241fcba45f0/react4xp.properties).
- You can make your own `webpack.overrides.js` file (or whatever name) that modifies the default webpack config, and refer to that file with the `overrideComponentWebpack` attribute in `react4xp.properties`. 

So basically, you should be able to just bump the packages in package.json, move your override values to `react4xp.properties`, overwrite your own `build.gradle` with the new version, and do something like this:

_webpack.config.overrides.js:_

```
module.exports = function(env, config) {
    config.resolve.symlinks = false;
    return config;
};
```

_react4xp.properties:_

```
...
overrideComponentWebpack = webpack.config.overrides.js
...
```

Tell me if it works! :)
